src/serializers/decoders/current/v16/decode_graph_entities.c

```
@@ -0,0 +1,496 @@
+	uint arrayLen = SerializerIO_ReadUnsigned(rdb);
+	SIValue list = SI_Array(arrayLen);
+	for(uint i = 0; i < arrayLen; i++) {
+		SIValue elem = _RdbLoadSIValue(rdb);
+		SIArray_Append(&list, elem);
+		SIValue_Free(elem);
+	}
+	return list;
+}
+
+static SIValue _RdbLoadVector
+(
```

cppcheck

[error] Memory leak

(memleak)